pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -592,6 +623,33 @@ func (p *criStatsProvider) addProcessStats(
 	ps.ProcessStats = mergeProcessStats(ps.ProcessStats, processStats)
 }
 
+func makePSIStats(stats *runtimeapi.PsiStats) *cadvisorapi.PSIStats {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.KubeletPSI) {
+		return nil
+	}
+	if stats == nil {
+		return nil
+	}
+	result := &cadvisorapi.PSIStats{}
```

Comment:
more of what I was suggesting is we update the CRI API to collect PSI stats instead of always falling back to cadvisor

Reply:
oh I see sorry I didn't understand we're converting from cri stats to cadvisor